BootloaderCorePkg/Tools/GenContainer.py

```diff
@@ -732,9 +734,9 @@ def main():
     cmd_display.add_argument('-f',  dest='comp_file',  type=str, required=True, help='Component input file path')
     cmd_display.add_argument('-o',  dest='sign_file',  type=str, default='', help='Signed output image name')
     cmd_display.add_argument('-c',  dest='compress', choices=['lz4', 'lzma', 'dummy'],  default='dummy', help='compression algorithm')
-    cmd_display.add_argument('-a',  dest='auth', choices=['rsa2048', 'sha256', 'none'], default='none',  help='authentication algorithm')
+    cmd_display.add_argument('-a',  dest='auth', choices=['SHA2_256', 'SHA2_384', 'RSA2048', 'RSA3072' 'none'], default='none',  help='authentication algorithm')
     cmd_display.add_argument('-k',  dest='key_file',  type=str, default='', help='Private key file path to sign component')
-    cmd_display.add_argument('-ht', dest='hash_type', type=str, default='SHA2_256', help='Hash Alg for signing')
+    cmd_display.add_argument('-ht', dest='hash_type', type=str,  choices=['SHA2_256', 'SHA2_384'], default='SHA2_256', help='Signing Hash Alg when auth type is RSA2048,RSA3072')

```

Comment:
To simplify the interface, prefer removing "-ht" later. when RSA2048 is selected, use SHA2_256, when RSA3072 is selected, use SHA2_384 inside.

Reply:
Incase user do not select hash option, may be we could default the hash options based on Sign method. 
RSA2048  - SHA256
RSA3072 - SHA384